File: Algorithms_Data_Structures/Task_Management_System/TaskTest.java
public class TaskTest {
    public static void main(String[] args) {
        TaskLinkedList pipeline = new TaskLinkedList();

        pipeline.addTask(new Task("T1", "Database Backup", "Pending"));
        pipeline.addTask(new Task("T2", "Code Review", "In-Progress"));
        pipeline.addTask(new Task("T3", "UI Deployment", "Pending"));

        pipeline.traverseTasks();

        System.out.println("\nSearching for T2:");
        System.out.println(pipeline.searchTask("T2"));

        System.out.println();
        pipeline.deleteTask("T2");
        pipeline.traverseTasks();
    }
}

File: Algorithms_Data_Structures/Task_Management_System/TaskTest.cs
using System;

public class TaskTest {
    public static void Main(string[] args) {
        TaskLinkedList pipeline = new TaskLinkedList();

        pipeline.AddTask(new Task("T1", "Database Backup", "Pending"));
        pipeline.AddTask(new Task("T2", "Code Review", "In-Progress"));
        pipeline.AddTask(new Task("T3", "UI Deployment", "Pending"));

        pipeline.TraverseTasks();

        Console.WriteLine("\nSearching for T2:");
        Console.WriteLine(pipeline.SearchTask("T2"));

        Console.WriteLine();
        pipeline.DeleteTask("T2");
        pipeline.TraverseTasks();
    }
}